team-validator.js

```diff
@@ -22,7 +22,11 @@ class Validator {
 		if (supplementaryBanlist && supplementaryBanlist.length) {
 			format = Object.assign({}, format);
 			if (format.banlistTable) delete format.banlistTable;
-			if (!format.banlist) format.banlist = [];
+			if (format.banlist) {
+				format.banlist = format.banlist.slice(0);
```

Comment:
If it's merged now, the burden of making it compatible with the refactor will fall on me... :|

...I guess that's mostly fine... Especially with how excited EG is about this.